src/backend/main.ts

```diff
@@ -1001,7 +1002,6 @@ ipcMain.handle(
       game.logFileLocation,
       'System Info:\n' +
         `${systemInfo}\n` +
-        '\n' +
```

Comment:
Won't this remove the empty line between the System Info & the settings? Right now, the log looks like this:
```
System Info:
...
GRAPHICS: GPU0: 3rd Gen Core processor Graphics Controller  VRAM: 256MB DRIVER:  
PROTOCOL: wayland

Game Settings: {
...
```
I'd say the empty line separates these two items quite well. Why remove it?

Reply:
In my GamesConfig folder, it showed two new lines. I see now this was from this line and the fact that I'm not on Linux
`${isLinux ? `PROTOCOL: ${xEnv}` : ''}``

I'll add it back